pkg/security/securitytest/BUILD.bazel

```diff
@@ -13,3 +14,16 @@ go_library(
         "@com_github_cockroachdb_errors//:errors",
     ],
 )
+
+bindata(
+    name = "embedded",
```

Comment:
This looks unused?

Reply:
nice catch! I just add `# keep` flag and the `bindata` within the main `go_library`